MANIFEST

```diff
@@ -1,19 +1,12 @@
-.appveyor.yml
-.gitignore
-.travis.yml
```

Comment:
These 3 are the only actually problematic ones, right?

Reply:
correct, I ve skipped some extra to avoid adding some noise to the MANIFEST
otherwise when generating it it s going to add some pollution

Reply:
These will probably be moved again once I rebuild the tarball. But it’s good for now. Thank you!